cli/src/cmd/cast/call.rs

```diff
@@ -31,13 +43,39 @@ pub struct CallArgs {
     )]
     data: Option<String>,
 
+    /// Forks the remote rpc, executes the transaction locally and prints a trace
+    #[clap(long, default_value_t = false)]
+    trace: bool,
+
+    /// Can only be used with "--trace"
+    ///
+    /// opens an interactive debugger
+    #[clap(long, requires = "trace")]
+    debug: bool,
+
+    /// Can only be used with "--trace"
+    ///
+    /// prints a more verbose trace
+    #[clap(long, requires = "trace")]
+    verbose: bool,
+
+    /// Can only be used with "--trace"
+    /// Labels to apply to the traces.
+    ///
+    /// Format: `address:label`
+    #[clap(long, requires = "trace")]
+    labels: Vec<String>,
+
+    /// Can only be used with "--trace"
+    #[clap(long)]
+    evm_version: Option<EvmVersion>,
```

Comment:
actually i think this is OK to use without trace i believe—so you can call into contracts using shanghai or another incompatible evm version

Reply:
cast call just does an rpc call so im not sure of any place to set a version

Reply:
well while not using `--trace`

Reply:
ah, fair enough! feel free to ignore then